test/core/V3/GenArt721CoreV3_ProjectConfigure.test.ts

```diff
@@ -18,6 +18,11 @@ import {
   advanceEVMByTime,
 } from "../../util/common";
 import { FOUR_WEEKS } from "../../util/constants";
+import {
+  SQUIGGLE_SCRIPT,
+  SKULPTUUR_SCRIPT_APPROX,
+  CONTRACT_SIZE_LIMIT_SCRIPT,
```

Comment:
Nit: I feel like we may want an assertion below that asserts that each of these imports is not the empty string, just to catch the case in the future where someone accidentally deletes those scripts or something but the above would still pass IIUC.

Reply:
This is addressed w/ https://github.com/ArtBlocks/artblocks-contracts/pull/299/commits/e71fae326601bcb009f9c03814e303ebfea765ab